bin/common/write_k8s_ip_to_environment.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o errexit
+while ! ifconfig eth1 | grep -q addr; do
+  sleep 3
+done
+K8S_VM_IP=$(ifconfig eth1 | grep -oE 'addr:[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | sed 's/addr://g')
```

Comment:
With both vagrant and CI deployments, there will be an eth1 which is the default route.

I can use `ip -4 addr show eth1 | awk '/inet/ { print $2 }' | cut -d/ -f1` to get the IP instead.